### PR TITLE
VET-105 Verify current frontend codebase

### PR DESCRIPTION
**Pull request by krawczykaghust**
The main goal was to adjust the current frontend base to be uniform

### CODE REVIEW

File: frontend/src/pages/Login.js [hunk 1]
```diff
@@ -15,15 +15,15 @@ const Login = () => {
...
-            <h3> Logowanie </h3>
+            <h3>Zaloguj się</h3>
 
-            <label> Email:  </label>
-            <input type = "email" onChange={(e) => setEmail(e.target.value)} value={email}/> 
+            <label>Email: </label>
+            <input type="email" onChange={(e) => setEmail(e.target.value)} value={email}/> 
 
-            <label> Hasło:  </label>
+            <label>Hasło:  </label>
```
Comment (maclipski) [inline frontend/src/pages/Login.js]: Double space after "Hasło:"

File: frontend/src/components/EquipmentForm.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import {useState} from "react"
-import {useEquipmentContext} from "../hooks/useEquipmentContext";
+import { useState } from "react"
+import { useEquipmentContext } from "../hooks/useEquipmentContext";
```
Comment (maclipski) [inline frontend/src/components/EquipmentForm.js]: is that ";" necessery? Haven't seen it in different files

File: frontend/src/views/ScheduleMainPage.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import {useEffect} from "react";
-import {useScheduleContext} from "../hooks/useScheduleContext";
+import { useEffect } from "react";
+import { useScheduleContext } from "../hooks/useScheduleContext";
 import ScheduleDetails from '../components/ScheduleDetails'
 import ScheduleForm from "../components/ScheduleForm";
```
Comment (maclipski) [inline frontend/src/views/ScheduleMainPage.js]: please keep ';' consistent

**Review by maclipski — CHANGES_REQUESTED** [review]
Minor style changes, code clear and acceptable

File: frontend/src/components/EquipmentForm.js (hunk 2 above)
Comment (krawczykaghust) [reply frontend/src/components/EquipmentForm.js]: The semicolon is unnecessary. We should avoid using it if not needed.

**Review by maclipski — APPROVED** [review]
Found nothing more to improve. Ready to be merged

**Pull request merged**